multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -102,11 +102,22 @@ def zero_shot_image_classification():
 
 @automm_presets.register()
 def object_detection():
+    # TODO: add another presets for training detection models from scratch
     return {
         "model.names": ["mmdet_image"],
         "model.mmdet_image.checkpoint_name": "yolov3_mobilenetv2_320_300e_coco",
         "env.eval_batch_size_ratio": 1,
         "env.precision": 32,
+        "env.strategy": "ddp",  # TODO: support ddp_spawn for detection
+        "env.auto_select_gpus": False,  # Have to turn off for detection!
+        "optimization.learning_rate": 5e-5,
+        "optimization.lr_decay": 0.95,
+        "optimization.lr_mult": 100,
```

Comment:
We need to add these new detection-specific hyperparameters to https://auto.gluon.ai/stable/tutorials/multimodal/customization.html

Reply:
Those are not new hyperparameters but they are not shown there. I guess it's because previously we always used the default.

Reply:
Done.